src/syscheckd/src/create_db.c

```diff
@@ -785,32 +783,33 @@ void fim_check_db_state() {
     }
 
     json_event = cJSON_CreateObject();
-    cJSON_AddNumberToObject(json_event, "db_entry_file_limit", syscheck.db_entry_file_limit);
-    cJSON_AddNumberToObject(json_event, "file_count", nodes_count);
+    cJSON_AddNumberToObject(json_event, "entries_limit", nodes_limit);
+    cJSON_AddNumberToObject(json_event, "entries_count", nodes_count);
+    cJSON_AddStringToObject(json_event, "fim_db_table", table_name);
 
-    if (nodes_count >= syscheck.db_entry_file_limit) {
-        _db_state = FIM_STATE_DB_FULL;
-        mwarn(FIM_DB_FULL_ALERT);
+    if (nodes_count >= nodes_limit) {
+        *db_state = FIM_STATE_DB_FULL;
+        mwarn(FIM_DB_FULL_ALERT, table_name);
```

Comment:
Applying the change proposed in the log definitions.
```suggestion
        mwarn(FIM_DB_FULL_ALERT, strcmp(table_name, FIMDB_FILE_TABLE_NAME) ? FIM_DB_FULL_ALERT_FILE: FIM_DB_FULL_ALERT_REG);
```

Reply:
done